client/src/index.css

```diff
@@ -10,9 +10,11 @@
 
 .parent-center {
   display: flex;
+  margin: 0 auto;
```

Comment:
width properties could defnitely be more cleaned up in the future, but just making these look as close to mockups as I could for now.

Reply:
@justfrankie I think this introduced a bug where the container in the login page is no longer the full height of the page. Would you be ok with changing the height to 100vh? I tried it on my local machine and couldn't get rid of the scrollbar with overflow: hidden
I'll take a closer look when I have time, but otherwise LGTM